portfolio_2019-03-15.sql

```diff
@@ -0,0 +1,43 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4541
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 127.0.0.1 (MySQL 5.6.43)
+# Database: portfolio
+# Generation Time: 2019-03-15 15:59:27 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table portfolio
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `portfolio`;
+
+CREATE TABLE `portfolio` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `Title` mediumtext,
```

Comment:
1. Do you expect titles to be more than 255 characters? Why not use VARCHAR?
2. Field names are normaly all lowercase.